_posts/en/newsletters/2019-04-16-newsletter.md

```diff
@@ -0,0 +1,152 @@
+and the number of public keys (n).  So, for simplicity's sake, we'll
+just plot the sizes of legacy P2SH-multisig, wrapped P2SH-P2WSH
+multisig, and native P2WSH multisig for all the possible k-of-n multisig
+combinations (up to the maximum 15-of-15 supported by legacy P2SH).  We
+can see that switching to P2SH-P2WSH brings large savings.
+Further switching to native P2WSH brings an additional constant-sized
+savings (about 35 bytes) that can increase savings by up to 15% for
+small multisigs (but which is marginal for large multisigs).
+
+![Plot of multisig transaction sizes with P2SH, P2SH-P2WSH, and P2WSH](/img/posts/2019-04-segwit-multisig-size.png)
+
+This accounts for almost all scripts being used with legacy and wrapped
+addresses, which still represent a majority of all transaction data
+included in blocks.  The savings described above are reductions in how
+much space the transactions take up in blocks, meaning they're the
+amount that users will save directly in fees compared to sending less
+efficient transactions at the same feerate.
+
+However, the savings also represent the supply of additional block
+capacity made available when more efficient transactions are sent.  When
+demand for Bitcoin block space is constant or decreasing, that extra
+supply of space can allow the addition of one or more low-feerate
+transactions to a block that would otherwise not have included them.
+Since the goal when choosing a feerate is to pay the smallest fee that
+gets your transaction confirmed by a particular block, a lower bottom
+feerate can reduce the minimum necessary fee not just for the direct
+users of native segwit but for everyone who creates
+transactions---including wallets and services that support sending to
+bech32 addresses.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].  Note: all merges described for Bitcoin Core and LND
+are to their master development branches; some may also be backported to
+their pending releases.*
+
+- [Bitcoin Core #15711][] generates bech32 addresses by default in the
+  GUI.  The user can still generate a P2SH-wrapped segwit address by
+  unchecking a box on the Request Payment screen in case they need to
+  receive money from a service that doesn't yet provide bech32 sending
+  support.  The bitcoind default of generating P2SH-wrapped segwit
+  addresses is not changed.
+
+- [C-Lightning #2506][] adds a `min-capacity-sat` configuration
+  parameter to reject channel open requests below a certain value.  This
+  replaces a hardcoded minimum of 0.00001 BTC previously in the code.
+
+- [LND #2933][] adds a document describing LND's current [backup and recovery
+  options][lnd recover].
+
+- [LND #2908][] logs the reason the node forced-closed one of its
```

Comment:
Generally, I think that logging PRs don't make the cut for 'interesting for our readers'. Is there a reason that this is different?

Reply:
I think that's a good policy, and I'll remove this.  My motivation was purely, "oh, I'd use that".